version.toml

```diff
@@ -0,0 +1,4 @@
+# -----DO NOT MODIFY THIS FILE-----
+# This file should only be modified by bumpver, which should in turn only be ran
+# via a GH Action
+version = "0.1.7"
```

Comment:
I think in order to complete this setup, you need to have something like [this](https://github.com/AllenCell/allencell-ml-segmenter/blob/main/.github/workflows/publish.yaml) which watches for modifications to `version.toml` then builds and publishes if `version.toml` has been modified. You could also build and publish manually if you prefer

Reply:
Note that ours is currently set up to publish to artifactory, so yours might look more like [this](https://github.com/AllenCell/allencell-ml-segmenter/commit/a5cd3d7deb3dc200b0e0c8e79b7c2a936b3ec5c5)

Reply:
ah good point, I'll modify our publish.yml